src/components/dialogs/delete-device-dialog.tsx

```diff
@@ -0,0 +1,59 @@
+import { Device } from "../../generated/client";
+import strings from "../../localization/strings";
+import GenericDialog from "../generic/generic-dialog";
+import { TextField } from "@mui/material";
+import { ChangeEvent, useState } from "react";
+
+/**
+ * Components properties
+ */
+interface Props {
+  deviceToDelete?: Device;
+  onClose: () => void;
+  onDelete: (device: Device) => Promise<void>;
+}
+
+/**
+ * Delete Device Dialog component
```

Comment:
Missing @params docs

Reply:
We haven't documented the props passed for components.